### ISSUE TITLE
error log

### ISSUE THREAD

**Issue by wangweitung** [CzBiX/ob-sync-server#3]
```
INFO:     192.168.1.1:57699 - "POST /user/info HTTP/1.1" 200 OK
INFO:     192.168.1.1:57713 - "OPTIONS /user/signin HTTP/1.1" 200 OK
INFO:     192.168.1.1:57713 - "POST /user/signin HTTP/1.1" 200 OK
INFO:     192.168.1.1:57713 - "POST /user/info HTTP/1.1" 200 OK
INFO:     192.168.1.1:57713 - "OPTIONS /vault/list HTTP/1.1" 200 OK
INFO:     192.168.1.1:57713 - "POST /vault/list HTTP/1.1" 200 OK
INFO:     192.168.1.1:57713 - "OPTIONS /subscription/list HTTP/1.1" 200 OK
INFO:     192.168.1.1:57724 - "POST /subscription/list HTTP/1.1" 200 OK
INFO:     ('192.168.1.1', 57726) - "WebSocket /sync" [accepted]
INFO:     connection open
websocket error
Traceback (most recent call last):
  File "/app/src/routers/sync.py", line 36, in websocket
    await conn.loop()
  File "/app/src/routers/sync.py", line 216, in loop
    await self.handle(msg)
  File "/app/src/routers/sync.py", line 360, in handle
    await self.on_push(msg)
  File "/app/src/routers/sync.py", line 240, in on_push
    if pieces and not self.vault.hash_exists(msg['hash']):
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/routers/sync.py", line 116, in hash_exists
    return dao.Vault.get_hash_count(self.db, self.vault_id, hash) > 0
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/dao.py", line 76, in get_hash_count
    count = len(db.exec(select(model.DocumentRecord).where(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: object of type 'ScalarResult' has no len()
INFO:     connection closed
INFO:     ('192.168.1.1', 57980) - "WebSocket /sync" [accepted]
INFO:     connection open
websocket error
Traceback (most recent call last):
  File "/app/src/routers/sync.py", line 36, in websocket
    await conn.loop()
  File "/app/src/routers/sync.py", line 216, in loop
    await self.handle(msg)
  File "/app/src/routers/sync.py", line 360, in handle
    await self.on_push(msg)
  File "/app/src/routers/sync.py", line 240, in on_push
    if pieces and not self.vault.hash_exists(msg['hash']):
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/routers/sync.py", line 116, in hash_exists
    return dao.Vault.get_hash_count(self.db, self.vault_id, hash) > 0
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/dao.py", line 76, in get_hash_count
    count = len(db.exec(select(model.DocumentRecord).where(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: object of type 'ScalarResult' has no len()
INFO:     connection closed
INFO:     ('192.168.1.1', 58156) - "WebSocket /sync" [accepted]
INFO:     connection open
websocket error
Traceback (most recent call last):
  File "/app/src/routers/sync.py", line 36, in websocket
    await conn.loop()
  File "/app/src/routers/sync.py", line 216, in loop
    await self.handle(msg)
  File "/app/src/routers/sync.py", line 360, in handle
    await self.on_push(msg)
  File "/app/src/routers/sync.py", line 240, in on_push
    if pieces and not self.vault.hash_exists(msg['hash']):
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/routers/sync.py", line 116, in hash_exists
    return dao.Vault.get_hash_count(self.db, self.vault_id, hash) > 0
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/dao.py", line 76, in get_hash_count
    count = len(db.exec(select(model.DocumentRecord).where(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: object of type 'ScalarResult' has no len()
```

**Comment by CzBiX** [comment]
thank you, fixed in [v0.2.5](https://github.com/CzBiX/ob-sync-server/releases/tag/v0.2.5)

**Comment by wangweitung** [comment]
> thank you, fixed in [v0.2.5](https://github.com/CzBiX/ob-sync-server/releases/tag/v0.2.5)

v0.2.5 ,it works fine,thanks~
